sdk/vision/azure-ai-vision-imageanalysis/README.md

```diff
@@ -226,7 +226,7 @@ To extract text for additional images, simply call the `analyze` multiple times.
 
 ### Extract text from an image URL
 
-This example is similar to the above, expect it calls the `analyze` method and provides a [publicly accessible image URL](https://aka.ms/azai/vision/image-analysis-sample.jpg) instead of a file name.
+This example is similar to the above, expect it calls the `analyze` method and provides a [publicly accessible image URL](https://aka.ms/azsdk/image-analysis/sample.jpg) instead of a file name.
```

Comment:
```suggestion
This example is similar to the above, except it calls the `analyze` method and provides a [publicly accessible image URL](https://aka.ms/azsdk/image-analysis/sample.jpg) instead of a file name.
```

Reply:
Good catch @srnagar, thank you, I missed this! fixed in latest commit.